index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="Styles.css">
```

Comment:
Good work adding link to html file

    Kindly remove "s" before the dote between Stlyes and css to match the one that you used as your file. So in this way the file will be correctly linked to html file 💯